### DISCUSSION TITLE
API for message deliveries parameter

### DISCUSSION THREAD

**Discussion by emazelmaurya** [postalserver/postal#2376] [Help with using Postal] [answered]
for accessing message deliveries API  I am sending the parameter '{id='2942'}' in JSON with URL and returns NULL, Please help me as I am new to POSTAL

**Comment by willpower232** [comment, marked as answer]
You need to post that JSON to /api/v1/messages/message (when authenticated with an API key)

![image](https://user-images.githubusercontent.com/1619102/225671890-7192ef68-2494-41e4-9f4e-8168eea86dd8.png)

**Reply by emazelmaurya** [reply]
@willpower232  
Thank you very much. Resolved!
